apps/src/aichat/views/AichatView.tsx

```diff
@@ -245,42 +256,20 @@ const AichatView: React.FunctionComponent = () => {
             headerContent={chatWorkspaceHeader}
             className={moduleStyles.panelContainer}
             headerClassName={moduleStyles.panelHeader}
-            rightHeaderContent={renderChatWorkspaceHeaderRight(() => {
-              dispatch(clearChatMessages());
-              analyticsReporter.sendEvent(
-                EVENTS.CHAT_ACTION,
-                {
-                  action: 'Clear chat history',
-                },
-                PLATFORMS.BOTH
-              );
-            })}
+            rightHeaderContent={
+              <div className={moduleStyles.chatHeaderRight}>
+                <CopyButton />
```

Comment:
Will we be implementing the overflow dropdown for the Copy action in time for the July pilot or leaving the 'Copy' button as is? @samantha-code